src/ToolBox/SOS/Strike/CMakeLists.txt

```diff
@@ -1,3 +1,11 @@
+# Set the RPATH of clrdbg so that it can find dependencies without needing to set LD_LIBRARY_PATH
```

Comment:
Did you really mean to say "clrdbg" in this comment or this is just a copy-and-paste error?